From d555635e42bc3bb05813568639d6d55bde02d2ea Mon Sep 17 00:00:00 2001
From: Jay Dobies <jason.dobies@redhat.com>
Date: Thu, 11 Jun 2015 14:06:17 -0400
Subject: [PATCH] Enable pacemaker by default for RDO Manager

Change-Id: Id0dc8833906fb36c4e0f57fdf617d20b6b7a42af
(cherry picked from downstream commit 54c32860897f15b3c1c7135476429ae939b2d59f)
Conflicts:
	overcloud-resource-registry-puppet.yaml
---
 overcloud-resource-registry-puppet.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/overcloud-resource-registry-puppet.yaml b/overcloud-resource-registry-puppet.yaml
index e829137..2a26f50 100644
--- a/overcloud-resource-registry-puppet.yaml
+++ b/overcloud-resource-registry-puppet.yaml
@@ -11,9 +11,9 @@ resource_registry:
   OS::TripleO::CephStorage: puppet/ceph-storage.yaml
   OS::TripleO::CephStorage::Net::SoftwareConfig: net-config-noop.yaml
   OS::TripleO::ControllerPostDeployment: puppet/controller-post.yaml
-  # set to controller-config-pacemaker.yaml to enable pacemaker
-  OS::TripleO::ControllerConfig: puppet/controller-config.yaml
   OS::TripleO::ComputePostDeployment: puppet/compute-post.yaml
+  # OS::TripleO::ControllerConfig: puppet/controller-config.yaml
+  OS::TripleO::ControllerConfig: puppet/controller-config-pacemaker.yaml
   OS::TripleO::ObjectStoragePostDeployment: puppet/swift-storage-post.yaml
   OS::TripleO::BlockStoragePostDeployment: puppet/cinder-storage-post.yaml
   OS::TripleO::CephStoragePostDeployment: puppet/ceph-storage-post.yaml
